desktop/cmp/rest/impl/RestFormModel.js

```diff
@@ -93,9 +100,11 @@ export class RestFormModel {
     //---------------------
     initForm(rec) {
         this.currentRecord = rec ? rec : {id: null};
-        this.isAdd = !rec;
+
+        this.isAdd = !rec || !rec.id;
```

Comment:
Would use `isNil` - not positive we have complete support for id:0 but no need to trip up here.

Reply:
Done.